### PR TITLE
Printing information about the running image (fix #131)

### LINKED ISSUE #131: Display info about scw-release

### PR DESCRIPTION
**Pull request by moul**

### CODE REVIEW

File: Linux/tree-common/init [hunk 1]
```diff
@@ -195,6 +195,19 @@ mountroot
...
 
+# Print rootfs information
+(
+    echo
+    if [ -r ${rootmnt}/etc/scw-release ]; then
+	. ${rootmnt}/etc/scw-release
+	esuccess "Image name:     ${IMAGE_DESCRIPTION}"
+	esuccess "Image release:  ${IMAGE_RELEASE}"
+    fi
+    esuccess "Usage on /:     $(df -h / | awk '/\// {print $(NF-1)}')"
```
Comment (QuentinPerez) [inline Linux/tree-common/init]: maybe `df -h ${rootmnt}` ?

POC:
```console
BusyBox v1.22.1
~ # df -h /
Filesystem                Size      Used Available Use% Mounted on
df: /: can't find mount point
~ # df -h /newroot/
Filesystem                Size      Used Available Use% Mounted on
/dev/nbd0                45.7G    421.2M     43.0G   1% /newroot
```

Comment (moul) [inline Linux/tree-common/init]: Yep :)

**Comment by moul** [comment]
fixed

**Comment by QuentinPerez** [comment]
LGTM

**Pull request merged**